examples/vault-cluster-enterprise/main.tf

```diff
@@ -0,0 +1,139 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A VAULT CLUSTER IN GOOGLE CLOUD
+# This is an example of how to use the vault-cluster to deploy a private Vault cluster in GCP with a Load Balancer in
+# front of it. This cluster uses Consul, running in a separate cluster, as its High Availability backend.
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "google" {
+  project = "${var.gcp_project}"
+  region  = "${var.gcp_region}"
+}
+
+# Use Terraform 0.10.x so that we can take advantage of Terraform GCP functionality as a separate provider via
+# https://github.com/terraform-providers/terraform-provider-google
+terraform {
+  required_version = ">= 0.10.3"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY THE VAULT SERVER CLUSTER
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "vault_cluster" {
```

Comment:
Note that in #25, it was pointed out that all our modules should expose their own `project_id` or `gcp_project_id` variables.

Reply:
okay, I can add that in, but you might want to merge #25 first.